### DISCUSSION TITLE
Production ready

### DISCUSSION THREAD

**Discussion by ronaldpanape** [kedacore/http-add-on#1029] [Q&A]
I am currently comparing knative to keda http add on, but not sure if keda is now production ready. Can someone verify this and elaborate on the comparisons between the 2

**Comment by vaibhavp3** [comment]
@ronaldpanape Did you get the confirmation on the Production ready status? Is still recommended for non-prods?

**Comment by linkvt** [comment]
We're currently actively working on the HTTP Add-on to make it more production ready, we just released v0.12.0 including some great improvements.
Right now I would say production readiness needs to be defined by your requirements and testing if it works well for you. If not, feel always free to report :) 

Closing this discussion as I'm doing a cleanup right now, if this is a bug that still exists please create an issue, otherwise just let us know.
